client/package.json

```diff
@@ -49,6 +49,7 @@
         "@ionic/vue-router": "^7.6.4",
         "@types/luxon": "^3.3.4",
         "axios": "^1.7.2",
+        "electron-builder-notarize": "^1.5.2",
```

Comment:
Should be a dev dependency. And I see `@types/luxon` in this list, it should also be a dev dependency.